src/iso15118/tbd_controller.cpp

```diff
@@ -28,8 +28,8 @@ void TbdController::loop() {
 
     if (not config.enable_sdp_server) {
         auto connection = std::make_unique<io::ConnectionPlain>(poll_manager, config.interface_name);
-        session =
-            std::make_unique<Session>(std::move(connection), std::move(d20::SessionConfig(evse_setup)), callbacks);
+        auto session_config = d20::SessionConfig(evse_setup);
+        session = std::make_unique<Session>(std::move(connection), std::move(session_config), callbacks);
```

Comment:
`d20::SessionConfig(evse_setup)` instead of `std::move(d20::SessionConfig(evse_setup))` should work here

Reply:
Thx! I changed that.